File: 210201/SW_Expert_1289_원재의메모리복구하기.java
package day0201;

import java.util.Scanner;

public class SW_Expert_1289_원재의메모리복구하기 {
public static void main(String[] args) {
	Scanner sc = new Scanner(System.in);
	
	int TC = sc.nextInt();
	int TCnt = 0;
	for (int tc =1; tc<=TC; tc++) {
		int cnt=0;
		TCnt++;
		String bit=sc.next();
		if(bit.charAt(0)=='1')cnt++;
		for (int i=0 ; i<bit.length()-1;i++) {
			if(bit.charAt(i)==bit.charAt(i+1)) {
				continue;
			}else {
				cnt++;
			}
		}
		System.out.println("#"+TCnt+" "+cnt);
		
	}
	

}
}

File: 210201/SW_Expert_1289_원재의 메모리 복구하기.py
T = int(input())
# 여러개의 테스트 케이스가 주어지므로, 각각을 처리합니다.
for test_case in range(1, T + 1):
    # ///////////////////////////////////////////////////////////////////////////////////
    cnt=0
    bit=input()

    if bit[0]=='1' :
        cnt+=1
    for x in range(len(bit)-1):
        if bit[x]==bit[x+1]:
            continue
        else:
            cnt+=1
    print("#"+str(test_case)+" "+str(cnt))